co2eq/purchase/index.js

```diff
@@ -105,25 +108,56 @@ export function modelCanRun(activity) {
 
   return false;
 }
+
 function correctWithParticipants(footprint, participants) {
   return footprint / (participants || 1);
 }
+
 function extractEur({ costAmount, costCurrency }) {
   return (costAmount && costCurrency)
     ? convertToEuro(costAmount, costCurrency)
     : null;
 }
 
+function CPIConversion(eurAmount, referenceYear, countryCodeISO2, datetime) {
+  if ((!eurAmount) || (!datetime)) {
+    return eurAmount;
+  }
+
+  const currentDateIndicator = datetime.getFullYear();
+
+  let CPIcurrent;
+  if (countryCodeISO2 && consumerPriceIndex[COUNTRY_CPI_INDICATOR][countryCodeISO2][currentDateIndicator]) {
+    CPIcurrent = consumerPriceIndex[COUNTRY_CPI_INDICATOR][countryCodeISO2][currentDateIndicator];
```

Comment:
What if we don't have a CPI for the `currentDateIndicator`? Raising an error would be my recommendation.